src/main/java/blackjack/domain/Participants.java

```diff
@@ -0,0 +1,62 @@
+package blackjack.domain;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Participants {
+    private final List<Playable> participants;
+
+    public Participants(Players players, Dealer dealer) {
+        participants = new ArrayList<>();
+        participants.add(dealer);
+        participants.addAll(players.values());
+    }
+
+    public String names() {
+        return participants.stream().filter(playable -> playable instanceof Player).map(Playable::getName).collect(Collectors.joining(", "));
```

Comment:
- `instanceof` 말고 다른 방법은 없을까요? :)
- stream 구문은 한줄에 하나씩 작성해주세요 ㅎㅎ
```suggestion
        return participants.stream()
                .filter(playable -> playable instanceof Player)
                .map(Playable::getName)
                .collect(Collectors.joining(", "));
```

Reply:
stream 줄바꿈 신경쓸게요!ㅎㅎㅎ 리팩토링 하면서 굳이 Participants 로 묶어줄 필요가 없을 것 같아 따로 받아주었습니다. 혹시 instanceof는 잘 사용되지 않나요?